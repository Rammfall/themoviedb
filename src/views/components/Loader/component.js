import React from 'react'
import { Col, Row, Spin } from 'antd'

const Loader = () => (
  <div className="top-margin">
    <Row
      type="flex"
      justify="center"
    >
      <Col>
        <Spin />
      </Col>
    </Row>
  </div>
)

export default Loader
