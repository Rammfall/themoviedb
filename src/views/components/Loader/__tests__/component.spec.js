import React from 'react'
import { shallow } from 'enzyme'

import Loader from '../component'

describe('MoviesLoader', () => {
  it('matches snapshot', () => {
    const component = shallow(<Loader />)

    expect(component).toMatchSnapshot()
  })
})
