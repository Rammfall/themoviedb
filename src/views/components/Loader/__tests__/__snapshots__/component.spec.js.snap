// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesLoader matches snapshot 1`] = `
<div
  className="top-margin"
>
  <Row
    justify="center"
    type="flex"
  >
    <Col>
      <Spin
        size="default"
        spinning={true}
        wrapperClassName=""
      />
    </Col>
  </Row>
</div>
`;
